docs/devel/api_changes.md

```diff
@@ -519,7 +519,7 @@ hack/update-codecgen.sh
 This section is under construction, as we make the tooling completely generic.
 
 At the moment, you'll have to make a new directory under `pkg/apis/`; copy the
-directory structure from `pkg/apis/extensions`. Add the new group/version to all
+directory structure from `pkg/apis/authentication`. Add the new group/version to all
```

Comment:
open to pointing to a different one, as long as it's one with the desired suffix

checked this group in particular, and opened https://github.com/kubernetes/kubernetes/pull/32409 to make sure it was added to the test and doc generation scripts as an example